cmd/sidekick-local.granica.ai-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
ummm ? should this be private

Reply:
Not necessarily -- these are just for serving https traffic locally. peep the PR description for motivations around this.

Reply:
I see makes sense